Root "yarn run"s that do "yarn workspaces run" don't work

For example, doing `yarn run lint-fix` in a fresh master gives the following error

```
$ yarn run lint-fix
yarn run v1.21.1
$ yarn workspaces run lint-fix

> @agoric/acorn-eventual-send
$ eslint --fix '**/*.{js,jsx}'
Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.

/Users/markmiller/src/ongithub/agoric/agoric-sdk/packages/acorn-eventual-send/index.js
  13:19  error  'options' is already declared in the upper scope  no-shadow

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /Users/markmiller/.nvm/versions/node/v11.13.0/bin/node
Arguments: /Users/markmiller/.nvm/versions/node/v11.13.0/lib/node_modules/yarn/lib/cli.js lint-fix
Directory: /Users/markmiller/src/ongithub/agoric/agoric-sdk/packages/acorn-eventual-send
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

--- a/.github/workflows/docker.yml
+++ b/.github/workflows/docker.yml
@@ -15,7 +15,9 @@ jobs:
     - name: Save SDK_VERSION
       run: echo "SDK_VERSION=$(jq -r .version package.json)" >> $GITHUB_ENV
     - name: Save MODDABLE_COMMIT_HASH
-      run: set $(git submodule status); echo "MODDABLE_COMMIT_HASH=$1" >> $GITHUB_ENV
+      run: |
+        set $(git submodule status packages/xsnap/moddable)
+        echo "MODDABLE_COMMIT_HASH=$1" >> $GITHUB_ENV
     - name: Build SDK image
       uses: elgohr/Publish-Docker-Github-Action@master
       with:

--- a/packages/captp/test/test-loopback.js
+++ b/packages/captp/test/test-loopback.js
@@ -1,3 +1,4 @@
+/* global setTimeout */
 import '@agoric/install-ses';
 import { Far } from '@agoric/marshal';
 import test from 'ava';

--- a/packages/cosmic-swingset/lib/ag-solo/fake-chain.js
+++ b/packages/cosmic-swingset/lib/ag-solo/fake-chain.js
@@ -148,7 +148,7 @@ export async function connectToFakeChain(basedir, GCI, delay, inbound) {
   let totalDeliveries = 0;
   async function deliver(newMessages, acknum) {
     totalDeliveries += 1;
-    console.info(`delivering to ${GCI} (trips=${totalDeliveries})`);
+    console.log(`delivering to ${GCI} (trips=${totalDeliveries})`);
 
     intoChain.push([newMessages, acknum]);
     if (!delay) {

